tlatools/org.lamport.tlatools/src/tla2sany/drivers/SANY.java

```diff
@@ -430,12 +449,18 @@ else if (args[i].toLowerCase().equals("-help")) {
            printUsage();
            System.exit(0);
       } else {
-           ToolIO.out.println("Command-line error: " + args[i]);
-           ToolIO.out.println("Use -help option for more information.");
+           ToolIO.out.println("Invalid option: " + args[i]);
+           ToolIO.out.println("Try 'SANY -help' for more information.");
            System.exit(-1);
```

Comment:
Same as above

Reply:
Ah, I guess this was already present. Well then we have the precedent of returning -1 on bad parameters so keep all of these in.